charts/rstudio-pm/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: rstudio-pm
 description: Official Helm chart for RStudio Package Manager
-version: 0.5.12
```

Comment:
Is this a breaking change? Otherwise bumping the minor version is probably not necessary?

Reply:
I assumed it may be since we're pointing to a different R version path? But I can change it. I'll follow your guys' lead for what constitutes a breaking change.

Reply:
It would be a breaking change if anybody is using the current image which does not have the symlink.

This was added in a PR that hasn't yet been merged into `main`
-  https://github.com/rstudio/rstudio-docker-products/pull/601